chaos/chaos.go

```diff
@@ -508,8 +508,8 @@ func testTwoClientsThatHaveReadLocksCrash(wg *sync.WaitGroup, ds *dsync.Dsync) {
 }
 
 type RWLocker interface {
-	Lock()
-	RLock()
+	Lock(id, source string)
```

Comment:
is source optional? but why?  - we can explain perhaps when someone looks at these arguments what should they provide as input. A test code showing this in dsync would be helpful

Reply:
Modified sample code in README.md and also `auth-locker` example to use `source` field instead of an empty string.